editor/src/components/canvas/ui/floating-insert-menu.tsx

```diff
@@ -160,26 +159,6 @@ function useGetInsertableComponents(): InsertableComponentFlatList {
   return insertableComponents
 }
 
-function getIsFlexBasedOnName_KILLME_EXPERIMENTAL(name: JSXElementName): boolean {
-  return (
-    name.propertyPath.propertyElements.length === 0 &&
-    (name.baseVariable === 'FlexRow' || name.baseVariable === 'FlexCol')
-  )
-}
-
-function getIsFlexDirectionBasedOnName_KILLME_SERIOUSLY_EXPERIMENTAL(
```

Comment:
https://github.com/concrete-utopia/utopia/issues/1580!

Reply:
yes, it's time to fix that :)